{"name": "test_login_without_password", "status": "failed", "statusDetails": {"message": "AssertionError: Status code 400 was expected and code 504 was returned.\nassert 504 == 400\n +  where 504 = <Response [504]>.status_code", "trace": "self = <test.test_CourierLogin.TestWithoutData object at 0x000001B2E717AC50>, base_url = 'http://qa-scooter.praktikum-services.ru/api/v1/'\n\n    def test_login_without_password(self, base_url):\n       json_data = {\n       \"login\": \"Vasiliy\"\n       }\n       response = requests.post(base_url + \"courier/login\", data=json_data)\n>      assert response.status_code == 400, f\"Status code 400 was expected and code {response.status_code} was returned.\"\nE      AssertionError: Status code 400 was expected and code 504 was returned.\nE      assert 504 == 400\nE       +  where 504 = <Response [504]>.status_code\n\ntest\\test_CourierLogin.py:29: AssertionError"}, "start": 1692259563237, "stop": 1692259623531, "uuid": "afaf059a-0e36-4f22-be27-07f57a95825e", "historyId": "46f10ba66826f7134aefca13aa8bfe9a", "testCaseId": "46f10ba66826f7134aefca13aa8bfe9a", "fullName": "test.test_CourierLogin.TestWithoutData#test_login_without_password", "labels": [{"name": "parentSuite", "value": "test"}, {"name": "suite", "value": "test_CourierLogin"}, {"name": "subSuite", "value": "TestWithoutData"}, {"name": "host", "value": "DESKTOP-36NVPK8"}, {"name": "thread", "value": "1756-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test.test_CourierLogin"}]}